arch/arm/configs/tsimx6_defconfig

```diff
@@ -1069,27 +906,27 @@ CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_ISO8859_15=m
 CONFIG_NLS_UTF8=y
-CONFIG_SECURITYFS=y
 CONFIG_CRYPTO_ECDH=y
 CONFIG_CRYPTO_CCM=y
 CONFIG_CRYPTO_GCM=y
-CONFIG_CRYPTO_CBC=y
 CONFIG_CRYPTO_CMAC=y
-CONFIG_CRYPTO_AES=y
-CONFIG_CRYPTO_LZO=m
+CONFIG_CRYPTO_DEFLATE=y
+CONFIG_CRYPTO_LZO=y
 CONFIG_CRYPTO_DEV_FSL_CAAM=y
+CONFIG_CRYPTO_DEV_SAHARA=y
+CONFIG_CRYPTO_DEV_MXS_DCP=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=m
 CONFIG_CMA_SIZE_MBYTES=64
 CONFIG_FONTS=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
-CONFIG_PRINTK_TIME=y
```

Comment:
Don't we want to leave printk time in on full defconfigs?

Reply:
Adding a requirement for this.